rate-providers/registry.json

```
@@ -107,6 +107,14 @@
                 }
             ]
         },
+        "0xA6aeD7922366611953546014A3f9e93f058756a2": {
+            "name": "QueenRateProvider",
+            "summary": "safe",
+            "review": "./QueenRateProvider.md",
+            "warnings": [],
```

We mention donation attacks in the report, so we should include the warning here.
```suggestion
            "warnings": [
                "donation"
            ],
```